readme (#7)
vtex-apps/list-wallet: docs/README.md

# The Wallet List Application

> The Wallet List app aims to create a front-end application to te store so that a list owner can add a value to the gift card from a credit generated by their gift list.

Reviewer: Just a suggestion:

```suggestion
The Wallet List app is a store-front application that allows a list owner to add credits to a gift card that has been previously generated by their gift list.
```

<img width="1440" alt="Captura de Tela 2022-02-08 às 17 25 59" src="https://user-images.githubusercontent.com/80836180/153070166-e0d67a93-8bb0-4b67-8377-30d9e5049a49.png">

## Installing

### The Store

To install the application on the store and thus be able to manage your wallet, you need to run the following command in your terminal

```json
  vtex install vtexbr.list-wallet@0.x
```
Once the app is installed, you can add the app to your store theme using the line

```json
  store-wallet-list 
```

## The Operation

### On Store

<img width="1440" alt="Captura de Tela 2022-02-08 às 17 25 59" src="https://user-images.githubusercontent.com/80836180/153070166-e0d67a93-8bb0-4b67-8377-30d9e5049a49.png">

> In the store it will be possible to check the credit that is still available to be added to the gift card (this amount is calculated from the difference between the balance of the lists and the amount of this balance that has already been added to the gift card) and the current balance of the list owner's gift card.

Reviewer: Another suggestion, please see if it makes sense 😅 :

```suggestion
In the store, the list owner can check the available credit which can be added to the gift card, and the current balance of their gift card. The available credit is calculated from the difference between the cash balance of the lists and the amount that has already been added to the gift card.
```

> In addition, it will be possible to make redemptions in the "Generate Redemption" field, where the redemption value must be within the range of credit available for this redemption. Finally, the gift card code will also be present, where it can be copied by clicking on the "Copy Code" button.

Reviewer: ```suggestion
In addition, it will be possible to make redemptions in the "Generate Redemption" field, as long as the input value is within the range of credit at hand for the redemption. Finally, the gift card code will also be available and can be copied by clicking on the "Copy Code" button.
```
